src/consensus/params.h

```diff
@@ -97,10 +123,13 @@ enum FundingStreamIndex : uint32_t {
     FS_ZIP214_BP,
     FS_ZIP214_ZF,
     FS_ZIP214_MG,
+    FS_ZIPTBD_LB,
     MAX_FUNDING_STREAMS,
 };
 const auto FIRST_FUNDING_STREAM = FS_ZIP214_BP;
 
+extern const struct FSInfo FundingStreamInfo[];
+
 enum FundingStreamError {
     CANOPY_NOT_ACTIVE,
     ILLEGAL_RANGE,
```

Comment:
`INSUFFICIENT_RECIPIENTS` on the line below.

Reply:
This variant is removed in https://github.com/zcash/zcash/pull/6912/commits/9ddc5742f224785f144d9dcc16a264b8bca7bebe